_includes/example.html

```diff
@@ -0,0 +1,21 @@
+{%- comment -%}
+Usage: {% include example.html content=markup %},
+where content is a capture with the HTML content
+id - null (default)
+class - "bd-example" (default)
+optional: hide_preview - disabled (default)
+optional: hide_markup - disabled (default)
+{%- endcomment -%}
+
+{%- assign preview_id = include.id -%}
+{%- assign preview_class = include.class | default: "bd-example" -%}
+
+{%- if include.hide_preview == null -%}
```

Comment:
OK, I think we should change the check to `if include.hide_preview != false`. Can you try it and change the two checks?

Reply:
Does not work, with `!= false`. Guess it does a [`not equals`](https://help.shopify.com/themes/liquid/basics/operators) check